src/logger/index.js

```diff
@@ -85,18 +92,23 @@ logger.onVersionNotChanged = (serviceId, type) => {
 
 logger.onRefilteringStarted = (numberOfServices, numberOfDocuments) => {
   logger.info(`👇  Refiltering ${numberOfDocuments} documents from ${numberOfServices} services…`);
+  recordedVersionsCount = 0;
 };
 
 logger.onRefilteringCompleted = (numberOfServices, numberOfDocuments) => {
-  logger.info(`👆  Examined ${numberOfDocuments} documents from ${numberOfServices} services for refiltering.\n`);
+  logger.info(`👆  Examined ${numberOfDocuments} documents from ${numberOfServices} services for refiltering.`);
+  logger.info(`    Recorded ${recordedVersionsCount} new versions.`);
 };
 
 logger.onTrackingStarted = (numberOfServices, numberOfDocuments) => {
   logger.info(`👇  Tracking changes of ${numberOfDocuments} documents from ${numberOfServices} services…`);
+  recordedSnapshotsCount = 0;
+  recordedVersionsCount = 0;
 };
 
 logger.onTrackingCompleted = (numberOfServices, numberOfDocuments) => {
-  logger.info(`👆  Tracked changes of ${numberOfDocuments} documents from ${numberOfServices} services.\n`);
+  logger.info(`👆  Tracked changes of ${numberOfDocuments} documents from ${numberOfServices} services.`);
+  logger.info(`    Recorded ${recordedSnapshotsCount} new snapshots and ${recordedVersionsCount} new versions.`);
```

Comment:
I'm in favor of removing spaces, like that:  
<img width="982" alt="Capture d’écran 2022-01-11 à 10 22 01" src="https://user-images.githubusercontent.com/1098708/148915715-92a4559d-ac63-45e6-a786-b30a9155e9b5.png">
If you agree, I already made the commit, I just have to push.

Reply:
Why not. I agree it would be nicer to have it aligned, but I don't like so much the idea of having a log line that contains only an emoji and that is not significant if taken away from its context 😕 Maybe we coud then use a different log level, like `debug`, for these lines?

Reply:
I'm in favor or removing the emoji and just adding a newline line after refiltering and tracking. @martinratinaud agree with that?

Reply:
Like that
<img width="967" alt="Capture d’écran 2022-01-12 à 09 24 21" src="https://user-images.githubusercontent.com/1098708/149090637-9a7b8779-2163-4648-a922-21009e85f272.png">

Reply:
Sure